doc/shell_long/README.md

```diff
@@ -0,0 +1,891 @@
+# 1. Test Objective  
+The document is to introduce how to test shell_long via CTP. It is similiar with shell test with CTP.For some shell cases, it spend so much time which affects the end of shell regression test, then remove them to shell_long test category.    
+
+# 2. Tools Introduction  

```

Comment:
As is:

# 2. Tools Introduction  

To be:

# 2. Test Introduction  
2.1 CTP Test Tool
2.2 Shell_long Test Cases